applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -1097,7 +1110,10 @@ Response syntax
 ~~~~~~~~~~~~~~~
 
 ::
+   .. option:: CONFIG_SLM_HTTPC - HTTP client support in SLM .. is enabled.
+   #XTCPSVR: <listen_socket_handle>,<income_socket_handle>,<data_mode>,<timeout>,<data_mode>,<family>
```

Comment:
Redundant option?

Reply:
**_.. option:: CONFIG_SLM_HTTPC - HTTP client support in SLM .. is disabled._** should be removed.
It might be something wrong when I port ipv6.